docs/topics/mta-what-is-the-toolkit.adoc

```diff
@@ -15,14 +15,16 @@
 [discrete]
 == What is the {ProductName}
 
-{ProductName} ({ProductShortName}) accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user or command line interfaces.
+{ProductName} ({ProductShortName}) accelerates large-scale application modernization efforts across hybrid cloud environments on {ocp-full} ({ocp-first}). This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift using the user interface or command line interface.

```

Comment:
ocp-full = `Red Hat OpenShift Container Platform`

ocp-first =  `{ocp-full} (RHOCP)`

If you looked at your preview, you would notice you have managed to create `Red Hat OpenShift Container Platform Red Hat OpenShift Container Platform  (RHCOP)`

You have to check your preview